### PR TITLE
implement team contacts

### PR DESCRIPTION
**Pull request by rocktavious**

### CODE REVIEW

File: opslevel/resource_opslevel_team_contact.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	if len(parts) == 2 {
+		d.Set("team", parts[0])
+		id = parts[1]
+		d.SetId(id)
+	}
+
+	identifier := d.Get("team").(string)
+	var err error
+	var team *opslevel.Team
+	if opslevel.IsID(identifier) {
```
Comment (rocktavious) [inline opslevel/resource_opslevel_team_contact.go:86]: I really hate this - we should probably push this complexity down into the opslevel-go library.  Not sure if now is the best time to do that though.

**Comment by rocktavious** [comment]
The submodule updates are for this - https://github.com/OpsLevel/opslevel-go/pull/55

File: opslevel/resource_opslevel_team_contact.go (hunk 1 above)
Comment (eochoa260) [reply opslevel/resource_opslevel_team_contact.go:86]: Ah yeah, that would be nice... I'm doing this in a few places now too.  Would it just involve reducing `GetTeam` and `GetTeamWithAlias` down to 1 function with this check in there?

Comment (rocktavious) [reply opslevel/resource_opslevel_team_contact.go:86]: Ya - we would need to adjust all our functions that take "graphql.ID" and instead take "IdentifierInput" and then make a different mutation weather its an Alias or an ID.  Its a pretty massive change so nows likely not the time.

**Pull request merged**